modules/benchmark/R/create.BRR.R

```diff
@@ -0,0 +1,54 @@
+##-------------------------------------------------------------------------------------------------#
+##' For each benchmark id, calculate metrics and update benchmarks_ensemble_scores
+##'  
+##' @name create.BRR
+##' @title Create benchmark reference run and ensemble
+##' @param ensemble.id id of ensemble run that will be used for benchmarking
+##' @param con database connection
+##' @export 
+##' 
+##' @author Betsy Cowdery 
+
+create.BRR <- function(ensemble.id, workflow, con){
+
+cnd1 <- workflow$hostname == fqdn() 
+cnd2 <- workflow$hostname == 'test-pecan.bu.edu' & fqdn() == 'pecan2.bu.edu'
+cnd3 <- workflow$hostname == 'pecan2.bu.edu' & fqdn() == 'test-pecan.bu.edu'
+  
+  
+if(cnd1|cnd2|cnd3){  # If the ensemble run was done on localhost, turn into a BRR
+  
+    BRR <- db.query(paste0("INSERT INTO reference_runs (model_id, settings, user_id, created_at, updated_at) VALUES(",workflow$model_id,", '",workflow$folder,"' , ",user_id,", NOW() , NOW()) RETURNING *;"),con)
```

Comment:
I don't see how this works -- settings is supposed to be a large text blob of PEcAn settings XML, but here you're just passing in a folder.